docs/forgeops_apply_apps.md

```diff
@@ -0,0 +1,60 @@
+## forgeops apply apps
+
+Deploy the ForgeRock apps (AM, IDM, UI)
+
+### Synopsis
+
+
+    Deploy the ForgeRock apps (AM, IDM, UI):
+    * Apply the ForgeRock apps
+    * use --tag to specify a different version to deploy
+
+```
+forgeops apply apps [flags]
+```
+
+### Examples
+
+```
+
+      # Deploy the ForgeRock apps listed in the "latest" release of Forgeops.
+      forgeops apply apps
+
+      # Deploy the ForgeRock apps listed in a specific release of Forgeops.
+      forgeops apply apps --tag 2020.10.28-AlSugoDiNoci
+```
+
+### Options
+
+```
+  -h, --help   help for apps
+```
+
+### Options inherited from parent commands
+
+```
+      --as string                      Username to impersonate for the operation
+      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+      --cache-dir string               Default cache directory (default "/home/jcastillo/.kube/cache")
+      --certificate-authority string   Path to a cert file for the certificate authority
+      --client-certificate string      Path to a client certificate file for TLS
+      --client-key string              Path to a client key file for TLS
+      --cluster string                 The name of the kubeconfig cluster to use
+      --context string                 The name of the kubeconfig context to use
+      --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
```

Comment:
I guess by default certificate is verified. 

Should we call it 'skip-tls-cert-verify'?

Reply:
these flags are inherited from a k8s library.